definitions/M3_E46.json

```diff
@@ -198,6 +199,124 @@
 			"max": 5,
 			"mul": 0.001221,
 			"hidden": true
-		}
+		}//, //Uncomment comma to the left and section below to enable CAN communication after connecting CAN Hi and Lo wires
+		/*{
+			"header": "Can button",
+			"canId": "0x329",
+			"offset": 3,
+			"hidden": false,
+			"noLog": true
+		},
+		{
+			"header": "Cruise active", //value 10 when icon lit on dash/active, value 2 when inactive/not lit
+			"canId": "0x545",
+			"offset": 0,
+			"hidden": false,
+			"noLog": true
+		},
+		{
+			"header": "Button press",
+			"unit": "b",
+			"expr": "buttonPress((x & 96) && !(y & 4))",
+			"x": "Can button", // if != 0 then button is active
+			"y": "Cruise active",
+			"hidden": true,
+			"noLog": true
+		},
+		{
+			"header": "Speedo",
+			"unit": "mph",
+			"canId": "0x153",
+			"offset": 1,
+			"length": 2,
+			"mul": 0.00477, //Use value .00769 for kph instead of mph. Assumes 24.72" diameter tires (245/40/17)
```

Comment:
Can we keep metric as default please? Just add comment with MPH value if possible please.